mail_mandrill/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_mail_mandrill_message_group_system","mail_mandrill_message group_system","model_mail_mandrill_message","base.group_system",1,1,1,1
+"access_mail_mandrill_event_group_system","mail_mandrill_event group_system","model_mail_mandrill_event","base.group_system",1,1,1,1
```

Comment:
This addon is a base addon for getting events from Mandrill. It only depends on mail addon and this is intentional.
There will be more addons that depends on this one and marketing (for instance) that creates security groups and menus in order to give other users access to this info.
BTW I've added read access to base.access_res_partner_group_user (Employee), as they have access for reading partners email addresses.